backend/database_handler/validators_registry.py

```diff
@@ -57,7 +57,7 @@ def create_validator(
             "stake": stake,
             "provider": provider,
             "model": model,
-            "config": json.dumps(config),
+            "config": AsIs("'" + json.dumps(config) + "'"),
```

Comment:
@bradleySuira why can't we just directly use
```suggestion
            "config": config,
```
? From [the docs](https://www.psycopg.org/docs/extras.html#adapt-json) seems like it automatically adapts it out of the box. If not, it suggests to use the [Json adapter](https://www.psycopg.org/docs/extras.html#psycopg2.extras.Json)

Reply:
thanks @AgustinRamiroDiaz used at the end `from psycopg2.extras import Json` and works well